docker/Dockerfile.indy

```diff
@@ -0,0 +1,260 @@
+ARG python_version=3.6.13
+ARG rust_version=1.46
+
+# This image could be replaced with an "indy" image from another repo,
+# such as the indy-sdk
+FROM rust:${rust_version}-slim-buster as indy-builder
+
+ARG user=indy
+ENV HOME="/home/$user"
+WORKDIR $HOME
+RUN mkdir -p .local/bin .local/etc .local/lib
+
+# Install environment
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+    automake \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    git \
+    libbz2-dev \
+    libffi-dev \
+    libgmp-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    libncursesw5-dev \
+    libsecp256k1-dev \
+    libsodium-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    libtool \
+    libzmq3-dev \
+    pkg-config \
+    zlib1g-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# set to --release for smaller, optimized library
+ARG indy_build_flags=--release
+
+ARG indy_version=1.16.0
+ARG indy_sdk_url=https://codeload.github.com/hyperledger/indy-sdk/tar.gz/refs/tags/v${indy_version}
+
+# make local libs and binaries accessible
+ENV PATH="$HOME/.local/bin:$PATH"
+ENV LIBRARY_PATH="$HOME/.local/lib:$LIBRARY_PATH"
+
+# Download and extract indy-sdk
+RUN mkdir indy-sdk && \
+    curl "${indy_sdk_url}" | tar -xz -C indy-sdk
+
+# Build and install indy-sdk
+WORKDIR $HOME/indy-sdk
+RUN cd indy-sdk*/libindy && \
+    cargo build ${indy_build_flags} && \
+    cp target/*/libindy.so "$HOME/.local/lib" && \
+    cargo clean
+
+# Package python3-indy
+RUN tar czvf ../python3-indy.tgz -C indy-sdk*/wrappers/python .
+
+# grab the latest sdk code for the postgres plug-in
+WORKDIR $HOME
+ARG indy_postgres_url=${indy_sdk_url}
+RUN mkdir indy-postgres && \
+    curl "${indy_postgres_url}" | tar -xz -C indy-postgres
```

Comment:
The `indy-sdk` is getting downloaded twice, once to build the `sdk` and once to build the `postgres plug-in`.  I'd recommend downloading the source once and building the both packages off the same source, like it's done in `von-image`.

Reply:
I think this is a fair simplification but to call out an implication: this would mean that the postgres storage plugin with a slightly different version from the Indy SDK could not be installed. I think I got the idea that this was desirable by looking at `von-image`'s build scripts, actually lol. So do we need to retain this ability?